### PR TITLE
chore: add Dropdown in storybook

### PR DESCRIPTION
**Pull request by gitstart-calcom**
## DEMO

https://www.loom.com/share/a33f7e06fcd34a4b9813b617f127c000?sid=b68bde67-ee50-42d9-9ae0-57fa1001dfd5

### CODE REVIEW

**Review by Udit-takkar — DISMISSED** [review]
@gitstart-calcom this is not how we implement and use dropdown.


you can check  here

Screenshot 2023-08-23 at 1 39 31 PM

https://github.com/calcom/cal.com/blob/376f023e0fd69c0f30569152785b56c277a32bc8/apps/web/components/eventtype/EventTypeSingleLayout.tsx#L337

**Comment by sean-brydon** [comment]
> @gitstart-calcom this is not how we implement and use dropdown.
> 
> you can check here
> 
> Screenshot 2023-08-23 at 1 39 31 PM
> https://github.com/calcom/cal.com/blob/376f023e0fd69c0f30569152785b56c277a32bc8/apps/web/components/eventtype/EventTypeSingleLayout.tsx#L337

Damn we really need to cleanup these old components :D

**Comment by gitstart-calcom** [comment]
Hi @Udit-takkar we made the requested changes here please take a look 😄

File: packages/ui/components/form/dropdown/dropdown.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+
+#### DropdownMenuContent
+
+<CustomArgsTable of={DropdownMenuContent} />
+
+#### DropdownMenuItem
+
+<CustomArgsTable of={DropdownMenuItem} />
+
+#### DropdownMenuSeparator
```
Comment (Udit-takkar) [inline packages/ui/components/form/dropdown/dropdown.stories.mdx:52]: also mention DropdownItem here

**Review by Udit-takkar — COMMENTED** [review]
LGTM. Left one comment

**Comment by gitstart-calcom** [comment]
@Udit-takkar We made the changes here please take a look 😄

**Review by PeerRich — APPROVED** [review]

**Pull request merged**